RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,240 @@
+
+## Activity Checklist
+- [ ] Prereqs to be performed prior to the CM starting:
+- [ ] Ensure that the cluster-state-service unit-tests pass.
+- [ ] Ensure that the cluster-state-service end-to-end tests pass.
+- [ ] Ensure that the daemon-scheduler unit-tests pass.
+- [ ] Ensure that the daemon-scheduler end-to-end tests pass.
+- [ ] Ensure that the daemon-scheduler integration tests pass.
+- [ ] Ensure that `deploy/docker/conf/docker-compose.yml` has been updated with the correct versions.
+- [ ] Ensure that `deploy/aws/conf/cloudformation_template.json` has been updated with the correct versions.
+- [ ] Ensure that `IamRoleInstance,IamRoleService,IamRoleTask,IamRoleLambda` in `deploy/aws/conf/cloudformation_template.json` have the correct role permissions.
+- [ ] Ensure that a $$blox_version$$ change log entry exists in `CHANGELOG.md`.
+- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/README.md` and `daemon-scheduler/README.md`.
+- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/VERSION` and `daemon-scheduler/VERSION`.
+- [ ] Ensure that the version is set to $$blox_version$$ in `cluster-state-service/versioning/version.go` and `daemon-scheduler/versioning/version.go`.
+- [ ] Ensure that the technician has a GitHub account and permissions to merge into the Blox master branch.
+- [ ] Ensure that the technician has a Docker Hub account and permissions to push into the bloxoss repositories.
+- [ ] Ensure that the technician has logged into Docker Hub via `docker login`.
+- [ ] CM activities:
+- [ ] Merge the GitHub pull request into the master branch.
+- [ ] Create a v$$blox_version$$ release tag in GitHub with commit $$github_hash$$.
+- [ ] Build the bloxoss/cluster-state-service:latest,$$blox_version$$,$$github_hash$$ images and publish to Docker Hub.
+- [ ] Build the bloxoss/daemon-scheduler:latest,$$blox_version$$,$$github_hash$$ images and publish to Docker Hub.
+
+## Rollback Procedure
+- Log into the Docker Hub console and delete the bloxoss/cluster-state-service:latest,$$blox_version$$,$$github_hash$$ images.
+- Log into the Docker Hub console and delete the bloxoss/daemon-scheduler:latest,$$blox_version$$,$$github_hash$$ images.
+- Publish the bloxoss/cluster-state-service:latest tag pointed to the previous release.
+- Publish the bloxoss/daemon-scheduler:latest tag pointed to the previous release.
+- Revert Blox repository changes.
+```
+$ cd <GitRepoBase>
+$ git revert v$$blox_version$$
+$ git push
+$ git fetch --tags origin
+$ git tag -d v$$blox_version$$
+$ git push origin :refs/tags/v$$blox_version$$
+```
+
+## Validation Activities
+
+#### Build the bloxoss/cluster-state-service:latest,$$blox_version$$,$$github_hash$$ images and publish to Docker Hub
+```
+$ cd <GitRepoBase>/cluster-state-service/
+$ make release
+$ docker tag bloxoss/cluster-state-service:latest bloxoss/cluster-state-service:$$blox_version$$
```

Comment:
Not a bad idea. Definitely outside the scope of this release, but something to consider in the future.